pages/installation/operating_system_packages.rst

```diff
@@ -131,5 +131,6 @@ Step-by-step guides
 Feedback
 --------
 
-Please open an `issue <https://github.com/Graylog2/fpm-recipes/issues>`_ in the `Github repository <https://github.com/Graylog2/fpm-recipes>`_ if you
-run into any packaging related issues. **Thank you!**
+Please open an `issue in the Github repository for the operating system packages <https://github.com/Graylog2/fpm-recipes>`__ if you
+run into any packaging related issues. If this Documentation is confusing or left questions to you please open an `issue in the Github repository for the documentation <https://github.com/Graylog2/documentation/issues>`__.
```

Comment:
Lower case "documentation" (also in all following files of this change set).

Maybe reword to "If you found this documentation confusing or have more questions, please open an issue…"